src/components/Main.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import bmw from '../assets/images/bmw.png';
```

Comment:
Please, try to compress the image. 1.3MB is too heavy.
You can use this page to compress the image: https://tinypng.com/

Reply:
Hi Juan, is has been edited already.

Looking forward for your approval.

Thanks!

Reply:
Thanks! approving now!